code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -169,6 +169,11 @@
 	if(stat == CONSCIOUS)
 		if(milkable_reagents && prob(reagent_regen_chance_per_tick))
 			milkable_reagents.add_reagent(milktype, rand(min_reagent_regen_per_tick, max_reagent_regen_per_tick))
+	if(src.reagents.has_reagent(PHAZON) && milktype != PHAZON) //if you roll the 1 in around 540 chances, you deserve your fountain of infinite phazon, godspeed
+		var/list/blocked_chems = list(ADMINORDRAZINE, PROCIZINE)
+		milktype = pick((chemical_reagents_list - blocked_chems)) //paismoke reacts instantly inside the cow, so it just constantly makes a smoke cloud harmlessly
+		name = "[lowertext(milktype)] cow"
```

Comment:
:-1: should just be `cow`

Reply:
i want it to be somewhat obvious this is a special cow
the grug event showed that the medical cows were not obvious enough, and were immediately butchered for meat

Reply:
them changing colors should make it obvious enough shouldn't it